Reply/app/src/main/res/values/type.xml

```diff
@@ -81,4 +81,11 @@
         <item name="android:textColor">@color/color_on_surface_emphasis_high</item>
     </style>
 
+    <style name="TextAppearance.Reply.EmailSubjectTitle" parent="TextAppearance.Material3.BodyLarge">
```

Comment:
nit: i think we can remove "Title" from "EmailSubjectTitle". feels a little redundant and made me think twice.